engines/director/movie.cpp

```diff
@@ -87,6 +87,8 @@ Movie::Movie(Window *window) {
 	_timeOutKeyDown = true;
 	_timeOutMouse = true;
 	_timeOutPlay = false;
+
+	_isBeepOn = false;
```

Comment:
I thought that beep is on by default, not off. Could you please check with the original?

Reply:
According to the D4 Dictionary, beepOn is FALSE by default. I will check with basiliskII and update

Reply:
I tested the movie. beepOn is off by default in the original too